geoviews/plotting/bokeh/plot.py

```diff
@@ -96,6 +99,8 @@ def get_extents(self, element, ranges):
         set_extent method to project the extents to the
         Elements coordinate reference system.
         """
+        if self.is_global:
+            return (-20026376.39, -20048966.10, 20026376.39, 20048966.10)
```

Comment:
I don't quite get the reason for the is_global flag (some description in the PR would be useful!), but in any case, how does knowing it's a plot of the whole globe determine the range here?  Can't there be infinitely many plots of the whole globe, centered at different longitudes?

Reply:
It's modeled on the cartopy ``ax.set_global()`` method, which simply means "Set the extent of the Axes to the limits of the projection."

>Can't there be infinitely many plots of the whole globe, centered at different longitudes?

No, neither matplotlib nor currently bokeh support wrapping at arbitrary longitudes. I have no issue adding a ``central_longitude`` option at a later point but that will require a whole bunch more work (and I'm not yet sure that it's even possible for certain projections in matplotlib).

Reply:
Actually I take the part about matplotlib back, it is possible to set the ``central_longitude`` in matplotlib directly on the ``crs`` in some cases.